pd-peds/config.lua

```diff
@@ -1,5 +1,3 @@
 Config = {}
 
-Config.SteamIdList = {
-    "steam:id"
-}
\ No newline at end of file
+Config.DefaultPedPerm = "perm.pedchange"
```

Comment:
This perm needs to add from the server.cfg but it can also be executed while the server running.